src/main/php/PDepend/Report/Jdepend/Chart.php

```diff
@@ -250,4 +206,55 @@ function ($a, $b) {
         // Remove temp file
         unlink($temp);
     }
+
+    /**
+     * @return array<int, array<string, mixed>>
+     */
+    private function getItems()
+    {
+        $items = array();
+        foreach ($this->code as $namespace) {
+            if (!$namespace->isUserDefined()) {
+                continue;
+            }
+
+            $metrics = $this->analyzer->getStats($namespace);
+
+            if (count($metrics) === 0) {
+                continue;
+            }
+
+            $items[] = array(
+                'size'         =>  $metrics['cc'] + $metrics['ac'],
+                'abstraction'  =>  $metrics['a'],
+                'instability'  =>  $metrics['i'],
+                'distance'     =>  $metrics['d'],
+                'name'         =>  Utf8Util::ensureEncoding($namespace->getName())
+            );
+        }
+
+        // Sort items by size
+        usort(
+            $items,
+            function ($a, $b) {
+                return ($a['size'] - $b['size']);
+            }
+        );
+
+        if ($items) {
```

Comment:
$items is an array, it can't be falsy.
```suggestion
        if (count($items) > 0) {
```

Reply:
An empty array is falsy
https://www.php.net/manual/en/types.comparisons.php
https://3v4l.org/3quYS